auto-update-ssl-certificates/README.md

```diff
@@ -0,0 +1,34 @@
+# OpenFisca Operations
+
+## Auto-update ssl certificates
+
+### Cron
+
+Make sure you have the cerbot cron file in /etc/cron.d/certbot with this inside :
```

Comment:
Use backticks around paths like `/etc/cron.d/certbot`

Remove ' ' before `:` in English